ntbs-integration-tests/NotificationPages/ClinicalDetailsPageTests.cs

```diff
@@ -122,7 +126,8 @@ public async Task PostNotified_ReturnsPageWithDiseaseSiteRequiredError_IfDisease
         public async Task PostNotified_ReturnsPageWithDiagnosisDateRequiredError_IfDiagnosisDateNotSet()
         {
             // Arrange
-            var initialPage = await client.GetAsync(GetPageRouteForId(Utilities.NOTIFIED_ID));
+            var initialUrl = GetCurrentPathForId(Utilities.NOTIFIED_ID);

```

Comment:
For most of these the `url` is not specific to the initial page load, so I wouldn't hate the variables being called just `url`